src/libaktualizr/utilities/utils_test.cc

```diff
@@ -46,6 +46,23 @@ TEST(Utils, PrettyNameOk) {
   EXPECT_FALSE(PrettyNameOk("foo-bar-123&"));
 }
 
+TEST(Utils, parseJSON) {
+  // this should not cause valgrind warnings
```

Comment:
Agreed... what warnings might it throw?

Reply:
There were some uninitialized memory accesses until 1.8.2 (but not in the ancient version we were using).

See https://github.com/open-source-parsers/jsoncpp/pull/651 and https://github.com/open-source-parsers/jsoncpp/issues/578.